dts/arm/microchip/sam/sam_d5x_e5x/common/samd5xe5x.dtsi

```diff
@@ -42,12 +42,43 @@
 			      <0x40001400 0x20>, <0x40001c00 0x140>;
 			reg-names = "mclk", "oscctrl",
 				    "osc32kctrl", "gclk";
+			interrupts = <1 0>, <2 0>, <3 0>, <4 0>, <5 0>, <6 0>, <7 0>;
```

Comment:
I recommend add the `interrupt-names` property to have documented, at least, what are all those 8 interrupts.